src/pool/srv_target.c

```diff
@@ -278,15 +266,24 @@ pool_alloc_ref(void *key, unsigned int ksize, void *varg,
 		goto err_group;
 	}
 
+	collective_arg.pla_pool = pool;
+	collective_arg.pla_uuid = key;
+	collective_arg.pla_map_version = arg->pca_map_version;
+	rc = dss_thread_collective(pool_child_add_one, &collective_arg, 0,
+				   DSS_ULT_IO);
+	if (rc != 0) {
+		D_ERROR(DF_UUID": failed to add ES pool caches: "DF_RC"\n",
+			DP_UUID(key), DP_RC(rc));
+		goto err_iv_ns;
+	}
+
 	*link = &pool->sp_entry;
```

Comment:
why move this part? for fixing bug?

Reply:
I think we should create IV namespace first, then create ds_pool_child which will start all containers and start aggregation ULT (where fetching snapshot list by IV).